charts/posthog/values.yaml

```diff
@@ -2221,6 +2225,21 @@ prometheus:
                 summary: Failed to process some session recording events and have been sent to the dead letter queue for review.
                 description: "The `session_recording_events_dlq` topic offset has increased over the past 5 minutes."
 
+            - alert: GraphileJobExecutionLag
+              expr: (max by(task_identifier) (posthog_celery_graphile_lag_seconds{task_identifier!="bufferJob"})) > 300
```

Comment:
Did we not get rid of the bufferJob ones yet?

Reply:
No, we have not deleted them yet.

Reply:
> and it's better to avoid us paging if we were to send more in the future.

paging yes, slack notification would be nice to have if we started adding these 🤷‍♀️